app/models/user.js

```diff
@@ -54,6 +54,11 @@ module.exports = (sequelize, DataTypes) => {
       tableName: 'users'
     }
   );
+  User.associate = models => {
+    User.hasMany(models.Weet, {
+      foreingKey: 'userId'
```

Comment:
The `foreignKey` should be added in the `belongsTo()` association in the `weet` model